### ISSUE TITLE
Where are stylelint errors supposed to appear?

### ISSUE THREAD

**Issue by stuartjnelson** [ManzDev/parcel-document-blank#1]
Hi @ManzDev,

Thanks for this repo its rad! I came across it while researching setting up my own parcel boilerplate.  With your setup and mine, I can't get Stylelint to work as I'd expect. I created a silly example below where I indent my styles by 1 and 4. I assumed I'd see linting errors in my console but I cannot. Where are the linting errors supposed to be displayed?

`.stylelintrc`
```
{
  "extends": "stylelint-config-standard",
  "rules": {
    "selector-nested-pattern": "^&",
    "indentation": 2,
    "no-descending-specificity": null,
    "no-eol-whitespace": null,
    "declaration-empty-line-before": null
  }
}
```

`index.pcss`
```
    .heading { color: red; }

 .copy { color: blue; }
```

**Comment by ManzDev** [comment]
Hi @stuartjnelson,

Very thanks! Using VSCode with plugin [Stylelint](https://marketplace.visualstudio.com/items?itemName=shinnn.stylelint) you can view errors in VSCode editor (more convenient / comfortable).

If you want view console errors you can use this command (change src\css\index.pcss for your path):
`npx stylelint src\css\index.pcss --config .stylelintrc`

You can add a *npm script* in `package.json` for fast run, inside "scripts" object 
`"lint": "stylelint src\css\index.pcss --config .stylelintrc"`

When run `npm run lint` you should see something like this:

![stylelint linter](https://user-images.githubusercontent.com/1507150/50254320-86794780-03e5-11e9-976d-40142ca62760.png)

Does this help you?

**Comment by stuartjnelson** [comment]
Hi @ManzDev,

Thanks for your detailed response and my apologies I didn't reply to you sooner! What I realized after submitting my ticket is that Parcel.js doesn't currently allow you to run it alongside being able to lint. Because of this, I had to ditch Parcel for Webpack as it's important for me to be able to have them in the console as well as in our editors to ensure everyone on the team can lint however they wish. 

On a separate note, I wonder if it's worth adding `"lint": "stylelint src\css\index.pcss --config .stylelintrc"` into the package.json in the project?
